README.md

```diff
@@ -117,6 +155,82 @@ The openHAB JSScripting runtime attempts to provide a familiar environment to Ja
 Scripts may include standard NPM based libraries by using CommonJS require.
 The library search path will look in `automation/js/node_modules` in the user configuration directory.
 
+### @runtime

```

Comment:
So , i think this is super helpful, but it's also a lot to put in the "Scripting Basics" section which was meant to be more of a quick overview of commonly used features.  Is there somewhere else we can put this farther down?  My fear is that new users to this library will get a little overwhelmed with the large table of runtime objects and miss the other basics .

Reply:
What about introducing a new section named like `Advanced Scripting` or `For Advanced Users` at the bottom of the README?

Reply:
I would respectfully disagree here.

As you know, the stuff from @runtime is available for basic use with the rules dsl and nashorn, and I would argue it is as commonly used/needed with ecmascript.

In my opinion you often need to access globals from there, and it is really hard to stay in pure js world without some sort of interaction with Java classes (QuantityType, ON, OFF, ZonedDateTime), would you not agree?

In fact, with my limited experience, you really need to pay attention to types since the comparisons work perhaps unlike one would expect (I would expect "dummy" == StringType("dummy") to equal True for example. Is this graalvm issue or just general javascript?)

Reply:
You are totally right that that part mentions important points, but personally I would still consider it as advanced as the normal user should use openhab-js and not directly access the Java parts.

I think I am more than just a normal user and I have plenty of rules in JavaScript and I do not use @runtime anywhere as I do not need to, instead I exclusively use openhab-js to interact with the openHAB runtime.

Reply:
https://github.com/openhab/openhab-js/pull/110/commits/667da0817b8d584c30347d30451b4dafeadd1fc9